src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module">
```

Comment:
I think it makes more sense to define this as a class specifically oriented around help, like `HelpHoverable`. We can consider generalizing this into a tooltip class that wraps sveltejs-tippy later if needed.

This would imply putting the help icon directly inside this component (only the text and links would be parametrized).